src/style.css

```diff
@@ -0,0 +1,25 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

Comment:
the last star shouldn't have margin-right

Reply:
done